src/tooltip/tooltip.component.ts

```diff
@@ -13,16 +16,14 @@ import { PopoverContainer } from "carbon-components-angular/popover";
 @Component({
 	selector: "ibm-tooltip",
 	template: `
-		<span
-			[attr.aria-labelledby]="this.isTemplate(this.description) ? id : null"
-			[attr.aria-describedby]="!this.isTemplate(this.description) ? id : null">
+		<span #contentWrapper>
 			<ng-content></ng-content>
 		</span>
```

Comment:
Why I removed the attributes? If I have a button as a child, it should be assigned the `aria` attribute not the span (wrapper). So if it is just an SVG for example, it does NOT need to have an aria attribute 🤔 atleast according to Accessibility checker